native/crypto/gcm_util.c

```diff
@@ -38,14 +38,13 @@ int Init_GCM(JNIEnv* env, jobject obj, jbyteArray key, jbyteArray iv, jint mode)
     return CRYPTO_FAILURE;
   }
 
-  jsize ivLength = (*env)->GetArrayLength(env, iv);
   jbyte* ivBytes = (*env)->GetByteArrayElements(env, iv, NULL);
   if (!ivBytes) {
     (*env)->ReleaseByteArrayElements(env, key, keyBytes, JNI_ABORT);
     return CRYPTO_FAILURE;
   }
 
-  GCM_JNI_CTX* ctx = Create_GCM_JNI_CTX(keyBytes, ivBytes, ivLength);
+  GCM_JNI_CTX* ctx = Create_GCM_JNI_CTX(keyBytes, ivBytes);
```

Comment:
we should check her that ivBytes is size GCM_IV_LENGTH_IN_BYTES, or fail, now that the other checking conditions are removed.